citi-onboarding/maracatronics
# Feature/robots section

## What i did
- Robots section creation;

## How to test
1. First step you have to use `git fetch --all` command on terminal;
2. Second change your branch to feature/robots-section with `git checkout feature/robots-section;`
3. After, open two terminals, one for the server and another for client and use `cd server` and `cd client` commands;
4. Then you have to use `nodemon index.js` on server terminal and `yarn/npm start` on client terminal;
5. In your browser open two pages: `http://localhost:3001/admin` and `http://localhost:3000/` ;
6. On Keystone page try to add some robots on 'Robos Combate', 'Robos Autonomos Individuai's and 'Robos Autonomos Coletivos' sections;
7. Reload `http://localhost:3000/ and` check if the robots was add;
8. Check the site responsivity. It's expected that with the screen decrease, the carousel adapts.
9. Also, check if the carousel is working.
